serverless.yml

```diff
@@ -46,6 +46,11 @@ package:
   individually: true
 
 resources:
+  Resources:
+    LambdaSg:
```

Comment:
@calebwsutton How does CloudFormation know in which VPC to place this security group?

Reply:
I see your point